std/array.d

```diff
@@ -380,6 +380,95 @@ if (isNarrowString!String)
     )));
 }
 
+/**
+ * Returns a static array from the stack with length of s and initializes
+ * it with copies of the elements of range $(D r). The remainder of the
+ * array is default-initialized.
+ *
+ * Params:
+ *      r = The range (or aggregate with $(D opApply) function) whose elements
+ *      are copied into the stack-allocated array. Must be as long or shorter
+ *      than s.
+ * Returns:
+ *      An initialized static array
+ */
+ForeachType!Range[s] array(size_t s, Range)(Range r)
+if (isInputRange!Range && !isInfinite!Range)
+{
+    alias E = ForeachType!Range;
+
+    import std.conv : emplaceRef;
+    import std.range : padRight;
+
+    if (__ctfe)
+    {
+        // Compile-time version to avoid memcpy calls.
+        Unqual!E[s] result;
+        size_t i;
+        foreach (e; r)
+        {
+
```

Comment:
huh?

Reply:
I copied that comment from regular `.array`. I think `emplaceRef` calls memCpy.